torch/_dynamo/output_graph.py

```diff
@@ -1459,6 +1502,14 @@ def compile_and_call_fx_graph(self, tx, rv, root):
             self.real_value_cache.clear()
 
             gm = _make_graph_module(root, self.graph)
+
+            # Saved tensors hooks are not used by the graph.
+            # GraphModule by default only copies used in the graph submodules.
+            # Copying them into the result graph manually.
+            if self.saved_tensors_hooks_subgraph_names:
```

Comment:
where is `self. saved_tensors_hooks_subgraph_names` getting set? (does this code ever run?)

Reply:
It is set at the end of `OutputGraph.__init__(...)`, output_graph.py:517
```
        self.saved_tensors_hooks_subgraph_names: Optional[list[str]] = (
            self.maybe_install_saved_tensors_hooks_subgraphs()
        )
```